src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      class="register-form"
+      >
+
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+  
+        <label for="surname" class="form-field">Surname:
+          <input 
+          type="text" 
+          name="surname" 
+          id="surname" 
+          autocomplete="off"
+          >
+        </label>
+     
+        <label for="name" class="form-field">Name:
+          <input 
+          type="text" 
+          name="name" 
+          id="name" 
+          autocomplete="off"
+          >
+        </label>
+  
+        <label for="age"
+         class="form-field"
+         >How old are You?
+          <input 
+          type="number" 
+          name="age" 
+          id="age"
+           min="12" 
+           max="100" 
+           value="12"
+          >
+        </label>
```

Comment:
```suggestion
        <label for="age" class="form-field">
          How old are You?
          <input 
            type="number" 
            name="age" 
            id="age"
            min="12" 
            max="100" 
            value="12"
          >
        </label>
```
fix code style like that

Reply:
for every label